js tests failed :(

```
  1 failing
  1) Debugger EvaluateXPath should correctly match xpath input:
     AssertionError: expected null to equal ''
      at http://localhost:8000/static/chai/chai.js:210
      at http://localhost:8000/static/chai/chai.js:2178
      at http://localhost:8000/static/cloudcare/js/form_entry/spec/debugger_spec.js:12
>> 1/44 tests failed (0.31s)
Warning: Task "mocha:cloudcare#form_entry" failed. Use --force to continue.
```

--- a/corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/debugger_spec.js
+++ b/corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/debugger_spec.js
@@ -9,13 +9,23 @@ describe('Debugger', function() {
                 result;
 
             result = evalXPath.matcher('', '');
-            assert.equal(result, '');
+            assert.equal(result, null);
 
+            // Should match /
             result = evalXPath.matcher('', '/data');
             assert.equal(result, '/data');
 
+            // Should not match parens
             result = evalXPath.matcher('', 'concat(');
-            assert.equal(result, '');
+            assert.equal(result, null);
+
+            // Should not match queries less than 1
+            result = evalXPath.matcher('', 'c');
+            assert.equal(result, null);
+
+            // Should match queries greater than 1
+            result = evalXPath.matcher('', 'co');
+            assert.equal(result, 'co');
         });
     });
 

